lib/lroPollStrategy.ts

```diff
@@ -157,6 +157,26 @@ export abstract class LROPollStrategy {
   }
 }
 
+function getOperationResponse(operationSpec: OperationSpec, response: HttpOperationResponse): OperationResponse | undefined {
+  const statusCode: number = response.status;
+  const operationResponses: { [statusCode: string]: OperationResponse } = operationSpec.responses;
+  let result: OperationResponse | undefined = operationResponses[statusCode];
+  if (!result) {
+    if (statusCode === 200) {
+      for (const expectedStatusCode of [201, 202]) {
```

Comment:
So..if a 200 came over on the wire, search for a response spec first for status 201, then status 202?

IMO with 2 elements this is simple enough to maybe just not be a loop.

`result = operationResponses[201] || operationResponses[202]`